compiler/test/compilable/importtests.d

```diff
@@ -0,0 +1,5 @@
+// https://issues.dlang.org/show_bug.cgi?id=23369
+import imports.aliasdump : iainBuclaw;
+import imports.aliasdump : iainBuclaw, iainBuclaw;
+
+import imports.aliasdump : aaa;
```

Comment:
This seems unrelated to this patch.

Reply:
It was to make sure there was no hiccups with the other forms